NorwegianRefugeeCouncil/core
# Auth screen

## What have you implemented?

* Previously the UI would flash before redirecting to Okta
* Add an endpoint that return 200 if authenticated and 401 if not
* In the frontend only render the UI if the endpoint returns a 200
* Redirect to okta on 401 (as previously implemented)
* Disable frontend e2e tests because the backend api wasn't being started. Previously this wouldn't cause issues, but with this pr the frontend won't load without a backend

## How to test it?

* Go incognito and go to core
* See Okta screen before seeing any core screen
* Login and see core
* Refresh/open core in new tab and go straight to core

## Extra/next steps

* Possibly loading screen when waiting on the auth endpoint
